napari/components/_layer_slicer.py

```diff
@@ -74,7 +75,7 @@ def __init__(self) -> None:
         """
         self.events = EmitterGroup(source=self, ready=Event)
         self._executor: Executor = ThreadPoolExecutor(max_workers=1)
-        self._force_sync = True
+        self._force_sync = get_settings().experimental.async_
```

Comment:
Right?

```suggestion
        self._force_sync = not get_settings().experimental.async_
```

Reply:
ah yes. You corrected this on my other PR and I didn't copy it over correctly. Doh!